### PR TITLE
Feature/#922 reduce chat memory consumption

### PR DESCRIPTION
**Pull request by micheljung**

### CODE REVIEW

File: .idea/dataSources.local.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <project version="4">
   <component name="dataSourceStorageLocal">
     <data-source name="faf@localhost" uuid="4818da61-e00f-4ebc-8ba7-90a6ce430cbf">
-      <database-info product="MySQL" version="5.7.18" jdbc-version="4.0" driver-name="MySQL Connector Java" driver-version="mysql-connector-java-5.1.45 ( Revision: 9131eefa398531c7dc98776e8a3fe839e544c5b2 )">
+      <database-info product="MySQL" version="5.7.18" jdbc-version="4.0" driver-name="MySQL Connector Java" driver-version="mysql-connector-java-5.1.45 ( Revision: 9131eefa398531c7dc98776e8a3fe839e544c5b2 )" family="MYSQL" exact-version="5.7.18">
```
Comment (1-alex98) [inline .idea/dataSources.local.xml]: I dont think that belongs here

File: .idea/misc.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
       <item index="2" class="java.lang.String" itemvalue="org.springframework.stereotype.Service" />
     </list>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_10" default="true" project-jdk-name="10" project-jdk-type="JavaSDK">
```
Comment (1-alex98) [inline .idea/misc.xml:10]: Did you mean updating to jdk_10

File: build.gradle [hunk 3]
```diff
@@ -308,16 +314,17 @@ dependencies {
...
   compile("com.github.Brutus5000:BiReUS-JCL:${bireusVersion}")
-//  compile("com.jfoenix:jfoenix:${jfoenixVersion}")
+  compile("com.jfoenix:jfoenix:${jfoenixVersion}")
+  compile("javax.annotation:javax.annotation-api:1.3.1")
 
   compile project(":webview-patch")
+  implementation 'org.javassist:javassist:3.23.1-GA'
 
-  compile("com.github.micheljung:JFoenix:6d00b59ca5327ca2754c63c94285b92a3badf7c5")
+//  compile("com.github.micheljung:JFoenix:6d00b59ca5327ca2754c63c94285b92a3badf7c5")
```
Comment (1-alex98) [inline build.gradle]: Lets clean this up then right

**Review by 1-alex98 — DISMISSED** [review]
To big to review all i better go and have a quick look around running it.

File: .idea/misc.xml (hunk 2 above)
Comment (micheljung) [reply .idea/misc.xml:10]: Yes

**Pull request merged**